yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1107,6 +1113,9 @@ func analyzeSchema() {
 	}
 
 	callhome.PackAndSendPayload(exportDir)
+
+	schemaAnalysisReport := createSchemaAnalysisReport(reportStruct)
```

Comment:
For consistency, `createSchemaAnalysisIterationCompletedEvent(reportStruct)`

Reply:
Done.